server/tests/routes/group.test.js

```diff
@@ -2,11 +2,27 @@ import request from 'supertest-as-promised';
 import httpStatus from 'http-status';
 import app from '../../../index';
 import { fixtures, build } from '../../utils/factories';
+import authService from '../../services/auth';
+import userService from '../../services/user';
 import User from '../../models/user';
 
 const groupUrl = '/api/v1/group';
 
 describe('## Group API', () => {
+  const user = fixtures.user();
+  let token;
+
+  beforeEach((done) => {
+    build('user', user)
+      .then((u) => {
+        authService.loginUser(u.email)
+          .then((authObject) => {
```

Comment:
Small detail here, but you could return the result from the `authService.loginUser(u.email)` so you don't start building a pyramid.

```
build('user', user)
  .then((u) => authService.loginUser(u.email))
  .then((authObject) => { ... }
```